### DISCUSSION TITLE
feat(Chart): 折线图线型设置

### DISCUSSION THREAD

**Discussion by DejanHuang** [dotnetcore/BootstrapBlazor#4506] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

请问如何自定义折线图的线型？通过AppendData将一个数组赋值给BorderDash的方式似乎不起作用

### Describe the solution you'd like

可自定义各数据集的线型

**Comment by bb-auto** [comment]
@huangyy21 Thank you for contacting us. We will give feedback later.

**Comment by ArgoZhang** [comment]
@huangyy21 请按照正确的格式填写问题。其提供复现示例，否则无法确定是那个环节出现问题，无法提供任何帮助
